### ISSUE TITLE
Multiple extension errors with Python 3.9 and GTK 4 installed

### ISSUE THREAD

**Issue by ahaasler** [Ulauncher/Ulauncher#724]
### Steps to reproduce
1. System with Ulauncher 5.10.0, Python 3.9 and GTK 4 installed.
2. Install https://github.com/Ulauncher/ulauncher-emoji
3. Save extension settings

### Current behavior
The extension is installed but not usable. The settings cannot be stored and the _em_ shortcut is not recognized. If you kill Ulauncher and start it again the extension appears as _**Crashed**_.

### Expected behavior
The extesion works

### Environment
**Ulauncher version**: 5.10.0
**Linux distro**: Archlinux
**Desktop environment**: Reproduced on two different PCs (Archlinux) with Sway 1.5.1 and Gnome 3.38

### Logs
https://gist.github.com/ahaasler/3f082b7487e273acc6e38ef7723ff5c3

### More information
Hi, I've been having multiple errors with extensions lately, new and already installed ones.

I don't really know when this started because I haven't used Ulauncher for a few months, but now I have Python 3.9 and GTK 4 installed alongside Ulauncher 5.10.0.

The error is first noticed when saving the extension settings and this error message appears:

Message: 'com.github.ulauncher.ulauncher-emoji'
Error Name: UnhandledError
Type: KeyError

Stacktrace:
```
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/ulauncher/ui/windows/PreferencesUlauncherDialog.py", line 198, in on_scheme_callback
    resp = rt.dispatch(self, scheme_request.get_uri())
  File "/usr/lib/python3.9/site-packages/ulauncher/utils/Router.py", line 49, in dispatch
    return callback(context, url_params)
  File "/usr/lib/python3.9/site-packages/ulauncher/ui/windows/PreferencesUlauncherDialog.py", line 444, in prefs_extension_update_prefs
    controller = ExtensionServer.get_instance().get_controller(ext_id)
  File "/usr/lib/python3.9/site-packages/ulauncher/api/server/ExtensionServer.py", line 76, in get_controller
    return self.controllers[extension_id]
KeyError: 'com.github.ulauncher.ulauncher-emoji'
```

I've searched for this error and it usually means that the extension needs for its dependencies to be manually installed, but the emoji extension doesn't have any.

Because I'm not familiar with Ulauncher code I tried directly executing the extension `main.py` and got this error:

```
/usr/lib/python3.9/site-packages/ulauncher/utils/display.py:2: PyGIWarning: Gdk was imported without specifying a version first. Use gi.require_version('Gdk', '4.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gdk, GdkX11
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/gi/importer.py", line 137, in load_module
    introspection_module = get_introspection_module(namespace)
  File "/usr/lib/python3.9/site-packages/gi/module.py", line 257, in get_introspection_module
    module = IntrospectionModule(namespace, version)
  File "/usr/lib/python3.9/site-packages/gi/module.py", line 109, in __init__
    repository.require(namespace, version)
gi.RepositoryError: Requiring namespace 'Gdk' version '3.0', but '4.0' is already loaded

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/adrian/.local/share/ulauncher/extensions/com.github.ulauncher.ulauncher-emoji/main.py", line 10, in <module>
    from ulauncher.api.shared.item.ExtensionResultItem import ExtensionResultItem
  File "/usr/lib/python3.9/site-packages/ulauncher/api/shared/item/ExtensionResultItem.py", line 6, in <module>
    from ulauncher.utils.image_loader import load_image
  File "/usr/lib/python3.9/site-packages/ulauncher/utils/image_loader.py", line 9, in <module>
    from gi.repository import Gtk, Gio, GLib, GdkPixbuf
  File "/usr/lib/python3.9/site-packages/gi/importer.py", line 139, in load_module
    raise ImportError(e)
ImportError: Requiring namespace 'Gdk' version '3.0', but '4.0' is already loaded
```

This led me to some changes made in Ulauncher (#458) to define GTK 3.0 to avoid this error, but adding the following lines in the extension code didn't fix anything.

```
import gi
gi.require_version('Gtk', '3.0')
```

After looking for differences with the extensions that did work (jetbrains for example) I found out that the ones failing didn't import `CopyToClipboardAction` or did it after `ExtensionResultItem`, which is the import that always seems to fail.

I've fixed the extensions that I use importing `CopyToClipboardAction` before `ExtensionResultItem` since no other thing I've tried works for me.

I've tried adding the following lines to all Ulauncher files that `import Gtk` without success:

```python
import gi
# pylint: disable=wrong-import-position
gi.require_version('Gtk', '3.0')
```

I wanted to make a PR fixing this myself but with the knowledge I currently have about Ulauncher I don't know where to continue. If you need any help or more info that I may have forgotten just comment.

**Comment by troycurtisjr** [comment]
I think this is a duplicate of #703

**Comment by ahaasler** [comment]
Yes, it appears to be the same one, I'll close this since a fix for #703 is already on dev. Thanks, I didn't see that one.
